lib/src/chart/radar_chart/radar_chart_painter.dart

```diff
@@ -68,6 +68,60 @@ class RadarChartPainter extends BaseChartPainter<RadarChartData> {
     drawDataSets(canvasWrapper, holder);
   }
 
+  @visibleForTesting
+  double getDefaultChartCenterValue() {
+    return 0;
+  }
+
+  double getChartCenterValue(RadarChartData data) {
+    final dataSetMaxValue = data.maxEntry.value;
+    final dataSetMinValue = data.minEntry.value;
+    final tickSpace = getSpaceBetweenTicks(data);
+    final centerValue = dataSetMinValue - tickSpace;
+
+    return dataSetMaxValue == dataSetMinValue
+        ? getDefaultChartCenterValue()
+        : centerValue;
+  }
+
+  @visibleForTesting
+  double getScaledPoint(RadarEntry point, double radius, RadarChartData data) {
+    final centerValue = getChartCenterValue(data);
+    final distanceFromPointToCenter = point.value - centerValue;
+    var distanceFromMaxToCenter = data.maxEntry.value - centerValue;
+
+    if (distanceFromMaxToCenter == 0) {
+      const preventDividingByZero = 0.001;
+      distanceFromMaxToCenter += preventDividingByZero;
```

Comment:
What if you return a value directly here? (instead of doing this trick)
I think you can return something like `0`